src/nvim/screen.c

```diff
@@ -4569,20 +4549,61 @@ static bool use_cursor_line_sign(win_T *wp, linenr_T lnum)
          && (wp->w_p_culopt_flags & CULOPT_NBR);
 }
 
+// Return true if CursorLine highlight is to be used.
+static bool use_cursor_line(win_T *wp, linenr_T lnum, int row, int startrow, int filler_lines)
```

Comment:
Splitting this into a separate function makes the comments below look strange.

Reply:
They look ok to me, or at least not any worse than before.

Can you suggest anything to make them less strange?

Reply:
Just do not split this into a separate function.